sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -825,7 +825,7 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
   def orc(paths: String*): DataFrame = format("orc").load(paths: _*)
 
   /**
-   * Returns the specified table as a `DataFrame`.
+   * Returns the specified table/view as a `DataFrame`.
```

Comment:
can we copy-paste the doc of `SparkSession.table` here?

Reply:
Thanks, done in c49c880